experimental/ComplexReflectionGroups/src/ComplexReflectionGroups.jl

```diff
@@ -0,0 +1,79 @@
+
+# A union type for all algebraic extensions of the field of rational numbers
+const QQAlgField = Union{NumField, QQField, QQBarField, QQAbField}
+const QQAlgFieldElem = Union{NumFieldElem, QQFieldElem, QQBarFieldElem, QQAbElem}
+
+# Imports (for stuff from experimental)
+import Oscar.LieAlgebras: coroot #no conflict, just same function name
+
+# Project files
+include("dual_vector_space.jl")
+include("is_root_of_unity.jl")
+include("hermitian_things.jl")
+
+include("ComplexReflection.jl")
+
+include("ComplexReflectionGroupType.jl")
+
+include("complex_reflection_group.jl")
+include("complex_reflection_group_LT.jl")
+include("complex_reflection_group_Magma.jl")
+include("complex_reflection_group_CHEVIE.jl")
+
+include("symplectic_reflection_group.jl")
+
+# Exports
```

Comment:
one generally wants to put the exports above the include statements. We had some very weird issues regarding that a few weeks ago (just with julia nightly, but anyway)